src/components/PurchaseSummary/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { PurchaseSummaryProps } from './type'
+import {
+  Container,
+  Title,
+  ItemContainer,
+  ItemText,
+  ProductsPrice,
+  DeliveryPrice,
+  ItemsTitle,
+  LineDivider,
+  Footer,
+  FooterItems,
+  FooterTitle,
+  PaymentInstallments,
+  Total,
+} from './style'
+
+import { PurchaseSummaryAccordion } from './PurchaseSummaryAccordion/index'
+
+const PurchaseSummary: React.FC<PurchaseSummaryProps> = ({
```

Comment:
remember to put intellisense in the component whenever you can, it is necessary to evaluate case by case but whenever possible, put intellisense